index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link href="https://fonts.googleapis.com/css?family=Crete+Round:regular,italic" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Poppins:100,100italic,200,200italic,300,300italic,regular,italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic" rel="stylesheet" />
+
+    <link rel="stylesheet" href="./styles.css">
+
+    <script src="https://kit.fontawesome.com/e971436820.js" crossorigin="anonymous"></script>
+
+    <title>Document</title>
+</head>
+<body>
+        <header class="nav-bar">
+            <h3>John Doe</h3>
+            <i class="fas fa-bars fa-lg"></i>
+        </header>
+        <section class="intro">
+            <div class="name">
+                <h1>Hey There.<br>I’m Alex</h1>
+            </div>
+            <div class="primary-content">
+                <h3>I’am a Software Developer</h3>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+            </div>
+            <div class="socials">
+                <a href=""><i class="fab fa-github"></i></a>
+                <a href=""><i class="fab fa-linkedin-in"></i></a>
+                <a href=""><i class="fab fa-angellist"></i></a>
+                <a href=""><i class="fab fa-twitter"></i></a>
+                <a href=""><i class="fab fa-medium-m"></i></a>
+            </div>
```

Comment:
- [OPTIONAL] 

- [ ]  Please consider using a `ul` tag to wrap all the social icons and use `li` tag to wrap each of the social icons instead of a `div` tag. This is because the social icons are unordered lists and therefore should be wrapped with a `ul` and `li` tag and also to improve the code structure. See my suggestion below:

```suggestion
            <ul class="socials">
              <li>
                <a href=""><i class="fab fa-github"></i></a>
              </li>
              <li>
                <a href=""><i class="fab fa-linkedin-in"></i></a>
              </li>
              <li>
                <a href=""><i class="fab fa-angellist"></i></a>
              </li>
              <li>
                <a href=""><i class="fab fa-twitter"></i></a>
              </li>
              <li>
                <a href=""><i class="fab fa-medium-m"></i></a>
              </li>
            </ul>
```